lib/phoenix_ecto/html.ex

```diff
@@ -3,12 +3,15 @@ if Code.ensure_loaded?(Phoenix.HTML) do
     def to_form(changeset, opts) do
       %{params: params, data: data} = changeset
       {name, opts} = Keyword.pop(opts, :as)
+      {id, opts} = Keyword.pop(opts, :id)
+
       name = to_string(name || form_for_name(data))
+      id = if is_nil(id), do: name, else: to_string(id) <> "_#{name}"
```

Comment:
I am thinking either `id || name`, without prefixing the `_#{name}` if an ID is given. WDYT?